quinn-udp/src/unix.rs

```diff
@@ -87,7 +92,7 @@ impl UdpSocketState {
         if is_ipv4 || !io.only_v6()? {
             if let Err(err) = set_socket_option(&*io, libc::IPPROTO_IP, libc::IP_RECVTOS, OPTION_ON)
             {
-                tracing::debug!("Ignoring error setting IP_RECVTOS on socket: {err:?}",);
+                debug!("Ignoring error setting IP_RECVTOS on socket: {err:?}",);
```

Comment:
IMO it would be nice to isolate these changes (that import the tracing event functions) in a separate commit from the introduction of the alternative impl.

Reply:
Sounds good. Extracted as a separate commit with https://github.com/quinn-rs/quinn/pull/1921/commits/fd4ede82da3669453aa97a4e580f7b75dc0e1537. Let me know if this is what you meant.